zerver/lib/markdown/help_settings_links.py

```diff
@@ -118,6 +118,11 @@ def getMarkdown(setting_type_name: str, setting_name: str, setting_link: str) ->
         setting_type_name=setting_type_name,
         setting_reference=setting_name,
     )
+    if setting_name == "Bots":
+        if setting_type_name == "Manage organization":
+            setting_type_name = "Organization settings"
```

Comment:
I'm not sure this makes sense -- the gear menu item is called "Manage organization" if you're an administrator, and "Organization settings" otherwise -- I'm not sure it makes sense to have this conditional naming for it? Like, if we called it "Manage organization" unconditionally, I think that seems fine -- administrators don't see the string "organization settings" in the app, and this is a feature documented as for administrators only...

Reply:
The thing is that without this conditional the instruction would say:
> Navigate to the **Bots** tab of the **Manage Organization** menu.

The reasoning for using "Organization settings" is to refer to the following part of the app:

<img width="1027" alt="image" src="https://user-images.githubusercontent.com/2343554/214171369-af581764-cf5f-4329-aa26-bbefea108462.png">

which does have the strings "Organization" and "Settings", and both administrators and users see the same strings.

Reply:
Yeah, I don't want to call the *menu* itself "Manage organization".

We should potentially add a word on an icon before "Settings" to make it more clear what you are looking at, in case you don't notice the tabs.

Reply:
I suppose we could also drop the special naming in the gear menu, and just call it "Organization settings".

Reply:
Yeah, just calling it "Organization settings" everywhere might be the simplest solution here. Worth discussing in #design; I think the "Manage organization" name was a slightly nice touch before we had any documentation, but it's super minor and creates this kind of awkwardness, especially now that we have roles like "moderator" that are kinda in between.

Reply:
https://chat.zulip.org/#narrow/stream/101-design/topic/.22Manage.20organization.22.20menu.20item/near/1496114